.github/workflows/tests.yml

```diff
@@ -46,7 +46,7 @@ jobs:
 
       - name: Install the project's dependencies
         shell: bash -el {0}
-        run: poetry install -v
+        run: poetry install --no-cache -v
```

Comment:
As of the time of this writing, `numpy=2.2.3` was released 11 minutes ago.

Locally, `poetry lock` found the updated numpy version and added it to the lock file.

However, on github actions, `poetry install` could not find the latest numpy version as it is too recent. Disabling cache seems to fix.